Cargo.toml

```diff
@@ -10,10 +11,11 @@ repository = "http://github.com/SSheldon/rust-objc"
 documentation = "http://ssheldon.github.io/rust-objc/objc/"
 license = "MIT"
 
-exclude = [".gitignore", "ios-tests/**", "xtests/**"]
+exclude = [".gitignore", "ios-tests/**", "xtests/**", ".travis.yml"]
 
 [features]
 exception = ["objc_exception"]
+gnustep_runtime = [ "gcc" ]
```

Comment:
Nitpick: I'd be okay with calling this feature just `gnustep`, it's a bit shorter but seems just as descriptive. Your thoughts?